f8a_worker/workers/githubstats.py

```diff
@@ -0,0 +1,55 @@
+from bs4 import BeautifulSoup
+import requests
+import logging
+from f8a_worker.base import BaseTask
+
+logger = logging.getLogger(__name__)
+
+class GithubStats(BaseTask):
+    """ Collects statistics by parsing Github Page """
+    _analysis_name = "github_manifest_details"
+
+    @classmethod
+    def _get_page(cls, repo_name):
+        github_url = 'https://github.com/'
+        url = github_url + "/" + repo_name
```

Comment:
`github_url` already ends with `/`

Also `github_url` is not used anywhere else so you can squash these two lines into `url = 'https://github.com/' + repo_name`

Reply:
removed the ending ```/``` form github_url and added with passed repo_name as .